.github/workflows/static_analysis.yaml

```diff
@@ -0,0 +1,62 @@
+name: Terraform Analysis
```

Comment:
can we have a comment / description about what this action is supposed to do?

Reply:
Added